### ISSUE TITLE
__mmc_blk_ioctl_cmd: data error -84 on dmesg and cid unchanged

### ISSUE THREAD

**Issue by marcus905** [raburton/evoplus_cid#12]
Hi,

I have a 2015 Samsung EVO SD card that the program tries to write to and ends with success, but the cid remains unchanged. Only indication of an error is the fact that there is a line in dmesg as the subject line (error -84 IIRC is Media Write Failed)

Do you have any information on that? Is there something else I can try?

Thank you!

**Comment by raburton** [comment]
The most likely explanation is that the card isn't compatible. I've not seen that error before though, so maybe it's worth trying it on a different device. It's not a USB reader is it? That won't work. Older laptops or android devices are the best bet.

**Comment by marcus905** [comment]
No, it's a Broadcom 57785 pcie reader that should be sdhci compliant. Still I'm unsure if it's the card, everything says it should be compatible but still...

I'll try with another PC, I hope I'm luckier then

**Comment by imic** [comment]
`<6>[ 1461.937954] power_debug_work_func_________over!
<6>[ 1461.951288] FG: ztemt_chg_print_info: bsoc=27 fgsoc=27 msoc=2663 T=345 mA=-1186 mV=4182 U=1 St=1
<3>[ 1462.440113] sdhci_msm 7864900.sdhci: mmc_blk_ioctl_cmd: data error -84
<3>[ 1464.098542] sdhci_msm 7864900.sdhci: mmc_blk_ioctl_cmd: data error -84
<3>[ 1465.034907] sdhci_msm 7864900.sdhci: mmc_blk_ioctl_cmd: data error -84`
